tests/testChain.cpp

```diff
@@ -0,0 +1,80 @@
+/* ----------------------------------------------------------------------------
+ * GTDynamics Copyright 2020, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * See LICENSE for the license information
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file  testChain.cpp
+ * @brief Test Chain class.
+ * @author: Dan Barladeanu, Frank Dellaert
+ */
+
+#include <CppUnitLite/TestHarness.h>
+
+#include <gtsam/geometry/Pose3.h>
+#include <gtsam/base/Matrix.h>
+
+#include "gtdynamics/dynamics/Chain.h"
+
+using namespace gtdynamics;
+using gtsam::Pose3;
+using gtsam::Rot3;
+using gtsam::Point3;
+using gtsam::Matrix;
+using gtsam::assert_equal;
+
+TEST(threeLinks, All) {
```

Comment:
Document tests (say what they test and how)

Reply:
added docs to tests
done